### PR TITLE
Consistency for operationId

### PR DESCRIPTION
**Pull request by GeoffreyFrogeye**
I encourage discussion on this one.

Basically what I did was:

1. Put every model name (Account, Currency...) in singular
2. Convert `getModelss` to `listModel` so no duplicate due to 1.
3. Open every shortcut (`getPiggiesByAccount` → `listPiggyBankByAccount`)

This will enable listing every action possible with a model, and figure out in a deterministic way what those actions are. This enable for greater re-usability in the client application. We can also do that with other things, but there are some limitations:

- API paths: they are hard to parse and sometimes not consistent, e.g. `/api/v1/attachments` and `/api/v1/attachment/{id}` (should I fill an issue for this btw?)
- tags: They're better off grouping requests by category in Swagger UI, and they don't provide enough informations by themselves unless you start to add a lot of them.

Also, this will add consistency with the schemas (e.g. what's the schema for `getAccount` ? You can either read `path['responses']['200']['content']['application/json']['schema']['data']` or just extract the `get` out of the `operationId` with this PR).

**However**, there is still be a consistency gap between the `type` field returned by the API, but to be consistent all the way with the [JSON:API spec](https://jsonapi.org/format/#document-top-level) we'd need to change the `operationId` fields and the schema names in the spec to `camel_case` and plural rather than `PascalCase` and singular. AFAIK there's nothing preventing that in the OpenAPI specification, it's only unusual. Does that sound good to you? If so, I would gladly do the renaming work (not that's a lot to do when you know how to use `sed` :smile: ).

### CODE REVIEW

**Comment by JC5** [comment]
Sounds great! What do you say, merge this one?

I can pick up the attachment thing on my side.

**Comment by GeoffreyFrogeye** [comment]
When you you said “Sounds great”, were you responding to the PR as a whole or specifically to the last paragraph where I suggested using the plural and camel_case for `operationId` and schema names?

Anyway you can merge this, even if I rename everything I'll do it after this and #23 to prevent too many merge conflicts.

**Comment by JC5** [comment]
Both actually. :+1: :grin:
